src/android_utils.py

```diff
@@ -224,6 +226,15 @@ def get_signature_key_issuing_organization():
     return value
 
 
+def get_dummy_user_name():
+    cache_key = "DUMMY_USER_NAME"
+    value = value_cache.get(cache_key)
+    if value is None:
+        value = _from_int(int(uuid4().hex[:8], 16)).replace("-", "_")
```

Comment:
Any reason to not import and use the [le_utils.proquint.generate](https://github.com/learningequality/le-utils/blob/cb33ad14da1f005efd21294878e1999f8ef267dd/le_utils/proquint.py#L78) function?

Reply:
Oh, originally, I was trying to fetch something meaningful from Android to feed in here rather than a UUID. So, no, no reason not to now.